### ISSUE TITLE
How to cleanup ignored words?

### ISSUE THREAD

**Issue by alexInhert** [codespell-project/codespell#2354]
There is ability to use `--ignore-words FILE` which ignores defined strings.
usually adding a string is done on need, the FILE tends to grow and grow and no one knows if the words in it are even needed any more.

Is there a way to say for example `--ignore-words-exact FILE` in which case if FILE contains a word that wasn't actually scanned the run will fail?

Example:

FILE contains:
```
ababa
fdindigd
```

Running over a string "Hello world ababa" with `--ignore-words` will be success
but with `--ignore-words-exact` it will fail until `fdindigd` is removed from FILE.

The goal is to keep FILE with exactly the entries that are currently being used.

**Comment by luzpaz** [comment]
This is an interesting idea. I think worthy of a feature request.

**Comment by peternewman** [comment]
Wouldn't it be more use, and generally more useful if there was a way to generate an ignore words file with codespell (or an ignore lines one). You could then just sort and diff the two (or that could potentially be built in). But it seems more useful to me, and also deals with most of the reporting stuff which would inevitably be the next request when it just fails if `fdindigd` is present in the file.

**Comment by vikivivi** [comment]
I would suggest rebuild from fresh. 
Run without `--ignore-words` and check what is failing. Then pipe all the failing words to rebuild the NEW-IGNORE-FILE.
Now you will have both the OLD and NEW IGNORE-FILE.

**Comment by alexInhert** [comment]
yes there are ways to achieve this with some workarounds but I think there is also justification to support it natively and easily with just adding the feature directly
